sdk/purview/azure-purview-sharing/tests/conftest.py

```diff
@@ -45,13 +45,23 @@ def add_sanitizers(test_proxy):
     tenant_id = os.environ.get("PURVIEWSHARING_TENANT_ID", "00000000-0000-0000-0000-000000000000")
     client_id = os.environ.get("PURVIEWSHARING_CLIENT_ID", "00000000-0000-0000-0000-000000000000")
     client_secret = os.environ.get("PURVIEWSHARING_CLIENT_SECRET", "00000000-0000-0000-0000-000000000000")
+    resource_group_name = os.environ.get("PURVIEWSHARING_RESOURCEGROUP", "fakeResourceGroup")
+    storage_account_provider = os.environ.get("PURVIEWSHARING_STORAGEACCOUNT_PROVIDER", "fakeStorageAccount")
+    storage_account_receiver = os.environ.get("PURVIEWSHARING_STORAGEACCOUNT_RECEIVER", "fakeStorageAccountR")
     add_general_regex_sanitizer(regex=subscription_id, value="00000000-0000-0000-0000-000000000000")
     add_general_regex_sanitizer(regex=tenant_id, value="00000000-0000-0000-0000-000000000000")
     add_general_regex_sanitizer(regex=client_id, value="00000000-0000-0000-0000-000000000000")
     add_general_regex_sanitizer(regex=client_secret, value="00000000-0000-0000-0000-000000000000")
+    add_general_regex_sanitizer(regex=resource_group_name, value="fakeResourceGroup")
+    add_general_regex_sanitizer(regex=storage_account_provider, value="fakeStorageAccount")
+    add_general_regex_sanitizer(regex=storage_account_receiver, value="fakeStorageAccountR")
     add_header_regex_sanitizer(key="Set-Cookie", value="[set-cookie;]")
     add_header_regex_sanitizer(key="Cookie", value="cookie;")
     add_body_key_sanitizer(json_path="$..access_token", value="access_token")
     add_body_key_sanitizer(json_path="$..atlasKafkaPrimaryEndpoint", value="000")
     add_body_key_sanitizer(json_path="$..atlasKafkaSecondaryEndpoint", value="000")
-    add_body_key_sanitizer(json_path="$..systemData.createdBy", value="000")
\ No newline at end of file
+    add_body_key_sanitizer(json_path="$..systemData.createdBy", value="000")
+
+@pytest.fixture(scope="session", autouse=True)
+def start_proxy(test_proxy):
+    return
```

Comment:
You can actually remove this fixture -- `add_sanitizers` will take care of starting up the test proxy since it requests the `test_proxy` fixture 🙂

Reply:
I've been wondering about that! There is no consistency across the other packages on using that or not, so I thought it wouldn't hurt back when I was trying to make sure test-proxy was set up correctly. I was actually hoping somebody would confirm whether or not it's needed, so thanks!